spec/models/user_spec.rb

```diff
@@ -1,5 +1,57 @@
 require 'rails_helper'
 
 RSpec.describe User, :type => :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  let(:user) { User.create(user_params)}
+  let(:user_params) { {
+                        :name => "Test User",
+                        :role => "admin reviewer editor",
+                        :email => "test@testuser.com",
+                        :institution => "Oregon State University",
+                        :password => "TestTest",
+                        :password_confirmation => "TestTest"
+                      }
+  }
+
+  context "When a user has all the roles" do
+    it "Should be truthy for all user role methods" do
+      expect(user.admin?).to be_truthy
+      expect(user.reviewer?).to be_truthy
+      expect(user.editor?).to be_truthy
+    end
+  end
+
+  context "When a user has the editor and reviewer role" do
+    let(:user_params) { {
+      :name => "Test User",
+      :role => "reviewer editor",
+      :email => "test@testuser.com",
+      :institution => "Oregon State University",
+      :password => "TestTest",
+      :password_confirmation => "TestTest"
+    }
+    }
+    it "Should be truthy for all user role methods" do
+      expect(user.admin?).to be_falsey
+      expect(user.reviewer?).to be_truthy
+      expect(user.editor?).to be_truthy
+    end
+  end
+
+  context "When a user has the editor and reviewer role" do
+    let(:user_params) { {
+      :name => "Test User",
+      :role => "editor",
+      :email => "test@testuser.com",
+      :institution => "Oregon State University",
+      :password => "TestTest",
+      :password_confirmation => "TestTest"
+    }
+    }
+    it "Should be truthy for all user role methods" do
```

Comment:
the test description reads as if each of these roles should be truthy, but only one is..  does the description need to be updated?